virtualbox/verify-install.sh

```diff
@@ -1 +1 @@
-test -d $HOME/Applications/"Virtualbox.app"
+test -d $HOME/Applications/"Virtualbox.app" || test -d $HOME/Downloads/"Virtualbox.app"
```

Comment:
have you checked this recently? i think that homebrew and casks just changed their default download location to `/Applications` which makes these checks easier, but invalidates all current checks